go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/onflow/atree v0.9.0
 	github.com/onflow/cadence v1.3.1
-	github.com/onflow/flow-go v0.38.1-0.20250213171922-77f4db56bb54
+	github.com/onflow/flow-go v0.38.1-0.20250218133611-bca6e7286932
```

Comment:
let's update this with the commit from master

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/760/commits/c0a8880c237b2e9848cb5a49666b9a0570f596cb .
Note that I used the merge commit from master, not the commit hash itself.
I also updated the `flow-emulator` dependency to the merge commit on master.